integration_tests/src/main/python/delta_lake_test.py

```diff
@@ -164,7 +164,7 @@ def convert_and_setup_name_mapping(spark):
 @allow_non_gpu(*delta_meta_allow)
 @delta_lake
 @ignore_order(local=True)
-@pytest.mark.skipif(not is_spark_340_or_later(), reason="Deletion Vectors only supported on Spark 3.4.0+")
```

Comment:
We still need to skip this test if OSS Spark version doesn't support deletion vectors, don't we?